test/framework/cloudstack.go

```diff
@@ -190,6 +190,26 @@ func WithCloudStackRedhat128() CloudStackOpt {
 	return withCloudStackKubeVersionAndOS(anywherev1.Kube128, RedHat8, nil)
 }
 
+// WithCloudStackRedhat9125 returns a function which can be invoked to configure the Cloudstack object to be compatible with K8s 1.25.
+func WithCloudStackRedhat9125() CloudStackOpt {
```

Comment:
Probably better to interpose the word `Kubernetes` here, else it reads as 9125, which is confusing.
```suggestion
func WithCloudStackRedhat9Kubernetes125() CloudStackOpt {
```

Reply:
Same for all the function names below.